pkg/crc/machine/start.go

```diff
@@ -784,6 +792,17 @@ func addNameServerToInstance(sshRunner *crcssh.Runner, ns string) error {
 	return nil
 }
 
+func enableEmergencyLogin(sshRunner *crcssh.Runner) error {
+	charset := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+	b := make([]byte, 8)
+	for i := range b {
+		b[i] = charset[rand.Intn(len(charset))] //nolint
+	}
+	logging.Infof("Emergency login password for core user is: %s", b)
```

Comment:
This will disappear from the log file when it is gets 'cleaned'/rotated.

Reply:
@gbraad updated.